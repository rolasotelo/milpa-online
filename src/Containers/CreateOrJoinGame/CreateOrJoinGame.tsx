import React from "react";

interface Props {}

const CreateOrJoinGame = (props: Props) => {
  return <div>MILPA</div>;
};

export default CreateOrJoinGame;
