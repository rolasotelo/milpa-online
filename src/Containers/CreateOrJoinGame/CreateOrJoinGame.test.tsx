import React from "react";
import renderer from "react-test-renderer";

import CreateOrJoinGame from "./CreateOrJoinGame";

describe("<CreateOrJoinGame />", () => {
  const defaultProps = {};
  const wrapper = renderer.create(<CreateOrJoinGame {...defaultProps} />);

  test("render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
