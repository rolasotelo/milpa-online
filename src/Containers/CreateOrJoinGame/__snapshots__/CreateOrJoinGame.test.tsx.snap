// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateOrJoinGame /> render 1`] = `
<div>
  MILPA
</div>
`;
